records/tools/uvision.uvproj.tmpl

```diff
@@ -72,6 +70,7 @@
             <UserProg2Name></UserProg2Name>
             <UserProg1Dos16Mode>0</UserProg1Dos16Mode>
             <UserProg2Dos16Mode>0</UserProg2Dos16Mode>
+            <nStopB1X>2</nStopB1X>
```

Comment:
I don't think uvision 4 respects the stop on error setting.  I can confirm that uvision 5 stops on an error correctly.  I think uv4 is just missing this feature.  I'm not sure if much can be done about this, just figured I would mention it.